lib/hashie/extensions/strict_key_access.rb

```diff
@@ -24,20 +24,24 @@ def initialize(msg = 'Setting or using a default with Hashie::Extensions::Strict
         end
       end
 
-      # NOTE: This extension would break the default behavior of Hash initialization:
+      # NOTE: Defaults don't make any sense with a StrictKeyAccess.
```

Comment:
I don't think it "breaks" this behavior as much as its incompatible (another word for "doesn't make sense"), maybe edit a bit.